data/json/deir-el-medina.json

```diff
@@ -0,0 +1,323 @@
+    "publications": [
+      {
+        "text": " Sandrine Vuilleumier, Laurence Cappa Dafflon, Paléographie hiéroglyphique du temple de Deir el-Médina, PalHier, Le Caire, Ifao, 2027. (in preparation)"
+      }
+    ],
+    "shortcode": "0853",
+    "spatialCoverage": [
+      {
+        "__type": "URL",
+        "type": "Geonames",
+        "url": "https://www.geonames.org/357994",
+        "text": "Egypt"
+      },
+      {
+        "__type": "URL",
+        "type": "Geonames",
+        "url": "http://www.geonames.org/347342",
+        "text": "Thebes"
+      }
+    ],
+    "startDate": "2021-02-01",
+    "temporalCoverage": [
+      {
+        "en": "300 BC"
+      },
+      {
+        "en": "AD 200"
+      }
+    ],
+    "url": {
+      "__type": "URL",
+      "type": "URL",
+      "url": "https://beyondthetext.ch/deir-el-medina",
+      "text": "External project website"
+    }
+  },
+  "datasets": [
+    {
+      "__id": "http://ns.dasch.swiss/repository#dsp-0853-dataset-000",
+      "__type": "Dataset",
+      "abstracts": [
+        {
+          "en": "The dataset consists of elements documenting the paleography of the hieroglyphs from the temple of Deir el-Medina. It includes images, orthophotos, DEMs, sections associated with external 3D models and drawings. It includes descriptive, analytical and bibliographical information relating to each sign associated with a hieroglyph."
+        }
+      ],
+      "alternativeTitles": [
+        {
+          "en": "Paléographie hiéroglyphique du temple ptolémaïque de Deir el-Medina"
+        },
+        {
+          "de": "Hieroglyphische Paläographie des ptolemäischen Tempels von Deir el-Medina"
+        }
+      ],
+      "accessConditions": "restricted",
+      "dateCreated": "2021-02-01",
+      "dateModified": "2027-12-31",
+      "datePublished": "2027-12-31",
+      "howToCite": "Deir el-Medina Hieroglyph Palaeography (2028). DaSCH. https://ark.dasch.swiss/ark:/72163/1/0853",
+      "languages": [
+        {
+          "en": "English"
+        },
+        {
+          "en": "French"
+        },
+        {
+          "en": "German"
+        }
+      ],
+      "licenses": [
+        {
+          "__type": "License",
+          "date": "2024-08-31",
+          "license": {
+            "__type": "URL",
+            "type": "Creative Commons",
+            "url": "https://creativecommons.org/licenses/by-nc-sa/4.0/",
+            "text": "CC BY-NC-SA 4.0"
+          }
+        }
+      ],
+      "attributions": [
+        {
+          "__type": "Attribution",
+          "agent": "http://ns.dasch.swiss/repository#dsp-0853-person-000",
+          "roles": [
+            "ProjectLeader",
+            "Creator"
+          ]
+        },
+        {
+          "__type": "Attribution",
+          "agent": "http://ns.dasch.swiss/repository#dsp-0853-person-001",
+          "roles": [
+            "Epigrapher",
+            "Creator"
+          ]
+        }
+      ],
+      "status": "Ongoing",
```

Comment:
You have used `Ongoing` with upper case "o", but the python script uses lower case. Have you verified that the status is displayed correctly?

Reply:
it is displayed ok. The schema suggests «Value should be one of: "In planning", "Ongoing", "On hold", "Finished"». I see though in older json files that is the "status" appears two times: once with uppercase and lowercase (with error)
![Screenshot 2024-09-02 at 14 58 24](https://github.com/user-attachments/assets/c5e55062-6142-45fc-95eb-8eead0186fde)

Reply:
Ok, great, thanks!  
I'll note this as something we should tidy up at some point - but of course that does npot have to happen in the scope of your PR.